### PR TITLE
Solved Visibility of showing papers while allotment

### LINKED ISSUE #135: Visibility of showing papers while allotment
The paper details are showing on the top of the page, user have to scroll up the page to see the papers for allotment. Fixing its visibility so that its easy to see the paper details is necessary

image

### PR DESCRIPTION
**Pull request by ygowthamr**
# Fix: Visibility of Paper Details During Allotment

## Description
This PR ensures paper details remain accessible during the allotment process by improving UI behavior.

Fixes #135

@Harshdev098 
Pls give me gssoc-extd,hacktoberfest-accepted,level2 for this pull request

### CODE REVIEW

**Comment by ygowthamr** [comment]
@Harshdev098 
Give me labels like gssoc-extd,hacktoberfest-accepted,level2 and merge my pr

**Review by Harshdev098 — CHANGES_REQUESTED** [review]
Don't just add the scrollTo method. If the allotment page become longer the user have to scroll down to see their left research papers
You can have a reference to #149

**Comment by ygowthamr** [comment]
@Harshdev098 
Ok, I will update it.

**Comment by ygowthamr** [comment]
@Harshdev098 
I have updated the code please check and merge my PR

File: public/script/paper_allotment.js [hunk 1]
```diff
@@ -50,6 +50,7 @@ const createListItem = (result, ul) => {
       display(result.id);
       const papers = document.querySelector(".papers");
       papers.style.opacity = "0.6";
+      window.scrollTo(0, 0);
```
Comment (Harshdev098) [inline public/script/paper_allotment.js:53]: remove this line because it scrolls the user to top

**Review by Harshdev098 — CHANGES_REQUESTED** [review]

File: public/script/paper_allotment.js (hunk 1 above)
Comment (ygowthamr) [reply public/script/paper_allotment.js]: done

**Pull request merged**